index.html

```diff
@@ -36,7 +36,255 @@ <h2 class="sd">
+            <li>Html</li>
+        </ul>
+    </div>
+    <div class="button-cont">
+        <button type="button" class="potfolio-but">See Project</button>
+    </div>
+
+    </div>
+</div>
+</section>
+
+<section>          
+    <div class="container-card">
+    <div class="placeholder">         
+    </div>
+    <div class="card-lists">
+        <div><h3>Multi-Post Stories Gain+Glory</h3></div>
+        <div class="items">
+        <ul>
+            <li>Ruby on rails</li>
+            <li>Css</li>
+            <li>Java script</li>
+            <li>Html</li>
+        </ul>
+    </div>
+    <div class="button-cont">
+        <button type="button" class="potfolio-but">See Project</button>
+    </div>
+
+    </div>
+</div>
+</section>
+
+<section>       
+    <div class="container-card">
+    <div class="placeholder">         
+    </div>
+    <div class="card-lists">
+        <div><h3>Multi-Post Stories Gain+Glory</h3></div>
+        <div class="items">
+        <ul>
+            <li>Ruby on rails</li>
+            <li>Css</li>
+            <li>Java script</li>
+            <li>Html</li>
+        </ul>
+    </div>
+    <div class="button-cont">
+        <button type="button" class="potfolio-but">See Project</button>
+    </div>
+
+    </div>
+</div>
+</section>
+          
+<section class="about-me">
+  <div> 
+      <div class="pic3" > 
+           <a href="#"><img src="./images/pointer.png" alt=""> </a>
+     <h1>About Me</h1>
+    </div>
+    <p> I can help you build a product , 
+          feature or website Look through 
+        some of my work and experience! If 
+           you like what you see and have a 
+          project you need coded, 
+              don’t hestiate to contact me.
+       </p>
+           </div>
+         <div class="bot-container">
+      <div class="bot-img">
+         <a href="#"><img src="./images/Group.png" alt=""></a>
+        </div>
+        <div class="botton-con">
+            <input type="button" value="Get My Resume" class="potfolio-but">
+        </div>
+       </div>
+                </section>  
+              
+                <section class="resume">  
+
+                    <div class="language-img">
+                        <a href="#"><img src="./images/icon-languages.png" alt="image"></a>
+                    </div>
+                    <p>Languages</p>
+                    <div class="resume-con">
+                            <ul>
+                                <li>Java script</li>
+                                <li>Ruby on rails</li>   
+                                <li>Html</li>
+                                <li>Css</li>
+                            </ul>
+                    </div>
+                </section>
+
+                <section class="resume">  
+                    <div class="language-img">
+                        <a href="#"><img src="./images/icon-frameworks.png" alt=""></a>
+                    </div>
+                    <p>Languages</p>
```

Comment:
- [ ] [OPTIONAL] I am including this as optional because the other reviewer forgot to mention it but I strongly suggest you implement it. Please make sure the technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`

Reply:
The language is a header not a list and will like to leave it that way as not to disorganized my entire work